### ISSUE TITLE
Not supported for Edge and Safari drivers

### ISSUE THREAD

**Issue by piyumiradeeshani** [sukgu/shadow-automation-selenium#73]
The current code is not supported for Edge and Safari drivers. Are there plans to implement this support in the near future?

**Comment by sukgu** [comment]
Yes, we are planning to implement this for other drivers also. Thanks for putting this as an enhancement. @piyumiradeeshani
